### PR TITLE
re-enable pull-kubernetes-bazel-build|test for release 1.6

### PR DESCRIPTION
**Pull request by BenTheElder**
see: https://github.com/kubernetes/test-infra/pull/4954

### CODE REVIEW

**Comment by BenTheElder** [comment]
build seems to be green: 
- https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53017/pull-kubernetes-bazel-build/8729/
test flaked(?) though, watching a rerun:
- https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53017/pull-kubernetes-bazel-test/9369/

File: prow/config.yaml [hunk 1]
```diff
@@ -988,6 +1128,103 @@ presubmits:
...
+    agent: kubernetes
+    context: pull-security-kubernetes-bazel-build
+    always_run: true
+    rerun_command: "/test pull-security-kubernetes-bazel-build"
+    trigger: "(?m)^/test( all| pull-security-kubernetes-bazel-build),?(\\s+|$)"
+    branches:
+    - release-1.6
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170929-ed26bd34-0.5.2
```
Comment (ixdy) [inline prow/config.yaml]: is this image too old? I thought we fixed this issue:
```
W1011 00:04:16.477] Error copying file:///workspace/k8s.io/kubernetes/_artifacts: CommandException: Encountered OSError running "file --mime-type /workspace/k8s.io/kubernetes/_artifacts/junit_bazel.xml"
```

Comment (BenTheElder) [reply prow/config.yaml]: Hmm, We should definitely make it match the others.

Comment (BenTheElder) [reply prow/config.yaml]: Actually it looks like they match already.

Comment (BenTheElder) [reply prow/config.yaml]: https://github.com/kubernetes/test-infra/blob/d6ec1d17746c28650a7f7c25f11461894d84e9ff/prow/config.yaml#L55-L67

**Pull request merged**